go.mod

```diff
@@ -75,3 +75,5 @@ require (
 	google.golang.org/protobuf v1.32.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
+
+replace github.com/heimweh/go-pagerduty => github.com/c-kaieong/go-pagerduty v0.0.0-20240321202501-cb1dc594997d
```

Comment:
**Reminder:** Drop replace and update `heimweh/go-pagerduty` once https://github.com/heimweh/go-pagerduty/pull/149 gets merged.

Reply:
Just to remind you ✌🏽 @c-kaieong https://github.com/heimweh/go-pagerduty/pull/149 has been merged, so We can move on with dropping this dependency replacement and dep udpate.